common/v2/components/AccountDropdown.tsx

```diff
@@ -19,14 +19,12 @@ interface IAccountDropdownProps {
 
 function AccountDropdown({ accounts, name, value, onSelect }: IAccountDropdownProps) {
   const { getAccountLabel } = useContext(AddressBookContext);
-  const relevantAccounts: StoreAccount[] = accounts
-    .filter(account => account.wallet !== WalletId.VIEW_ONLY)
```

Comment:
~~Shouldn't we keep the filter to avoid a VIEW_ONLY (an account that can't sign a message) from being selected in a send or sign flow?~~

Reply:
~~Or since we share it with receive, maybe adding a boolean prop to the component to choose if we filter or not?~~

Reply:
Filtering the accounts before passing them is even cleaner. Nice!